app/controllers/collection_drafts_controller.rb

```diff
@@ -403,4 +380,42 @@ def validate_picklists(errors, metadata)
   def set_language_codes
     @language_codes = cmr_client.get_language_codes.to_a
   end
+
+  def new_view_setup
```

Comment:
As this and the edit_view_setup are used twice, they have been extracted here.

Reply:
nice refactoring